### PR TITLE
add mw660 keyboard

### PR DESCRIPTION
**Pull request by TW59420**
## Description
MWStudio is a custom keyboard enthusiast
Upload qmk firmware of mw660 PCB

## Types of Changes
- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist
- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mwstudio/mw660/chconf.h [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/* Copyright 2022 TW59420 <https://github.com/TW59420>
```
Comment (zvecr) [inline keyboards/mwstudio/mw660/chconf.h:1]: This file should be removed

File: keyboards/mwstudio/mw660/mw660.c [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/* Copyright 2022 TW59420 <https://github.com/TW59420>
```
Comment (zvecr) [inline keyboards/mwstudio/mw660/mw660.c:1]: This file should be removed

File: keyboards/mwstudio/mw660/mw660.h [hunk 3]
```diff
@@ -0,0 +1,32 @@
+/* Copyright 2022 TW59420 <https://github.com/TW59420>
```
Comment (zvecr) [inline keyboards/mwstudio/mw660/mw660.h:1]: This file should be removed as info.json has matrix data

File: keyboards/mwstudio/mw660/rules.mk [hunk 4]
```diff
@@ -0,0 +1,4 @@
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+ENCODER_ENABLE = no
```
Comment (zvecr) [inline keyboards/mwstudio/mw660/rules.mk:4]:
```suggestion
# This file intentionally left blank
```

File: keyboards/mwstudio/mw660/info.json [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+    "keyboard_name": "MW660",
+    "maintainer": "TW59420",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
```
Comment (zvecr) [inline keyboards/mwstudio/mw660/info.json:12]:
```suggestion
        "nkro": true,
        "rgblight": true
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/mwstudio/mw660/readme.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+# MW660
+
+![MW660](https://i.imgur.com/kJx0mpi.png)
```
Comment (drashna) [inline keyboards/mwstudio/mw660/readme.md:3]: This uses a smaller but still high res image that is more suitable for github markdown.
```suggestion
![MW660](https://i.imgur.com/kJx0mpih.png)
```

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by TW59420** [comment]
@zvecr  Hi, Can you help me merge branches

**Pull request merged**